homeassistant/components/proximity/sensor.py

```diff
@@ -0,0 +1,120 @@
+"""Support for Proximity sensors."""
+
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import CONF_NAME, UnitOfLength
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import slugify
+
+from .const import ATTR_DIR_OF_TRAVEL, ATTR_DIST_TO, ATTR_NEAREST, DOMAIN
+from .coordinator import ProximityDataUpdateCoordinator
+
+SENSORS_PER_ENTITY: list[SensorEntityDescription] = [
+    SensorEntityDescription(
+        key=ATTR_DIST_TO,
+        device_class=SensorDeviceClass.DISTANCE,
+        native_unit_of_measurement=UnitOfLength.METERS,
+    ),
+    SensorEntityDescription(
+        key=ATTR_DIR_OF_TRAVEL,
+        translation_key=ATTR_DIR_OF_TRAVEL,
```

Comment:
Should this be an enum sensor?

Reply:
sorry, but i do not get it

Reply:
There's a sensor device class called `enum`.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
uhhh ... looks interesting ... I will rework this sensor 👍

Reply:
2b81bd7